src/test/java/hudson/maven/MavenBuildTest.java

```diff
@@ -380,4 +407,27 @@ public boolean projectActionsGotRequested() {
     private static class TestAction extends InvisibleAction{
     }
 
+    private static class FailingBuildWrapper extends BuildWrapper {
+        @Override
+        public Environment setUp(AbstractBuild build, Launcher launcher, BuildListener listener) throws IOException, InterruptedException {
+        	return null;
+        }
+    }
+
+    private static class TearingDownBuildWrapper extends BuildWrapper {
```

Comment:
I suppose it should be marked as <code>@TestExtension</code> to operate correctly. Both classes should be also public AFAIK